grafana/grafana
# sql: numeric inputs: use it's own simple implementation

(NOTE: the PR is not finished currently)

(part of https://github.com/grafana/grafana/issues/72892)

the shared sql support codebase imported a number-input-widget from `app/core/components/OptionsUI/NumberInput`, which is bad if we want to externalize the sql datasources.
we replaced it with using grafana-ui's input field directly (thanks @JoaoSilvaGrafana for the tip!)

how to test:
1. go to the config-page of an sql datasource plugin (mssql/mysql/postgres)
2. adjust the values in the "Connection Limits" section, and make sure they work.
3. choose one of the number fields in this section, and do this:
    1. set the value to `42`, press [save&test], then reload the page, verify that it is set to `42`.
    2. set the value to `4`, press [save&test], then reload the page, verify that it is set to `4`.
    3. set the value to empty, verify that this is possible to achive, then press [save&test], then reload the page, verify that it is still set to `4`.
